src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -172,7 +170,10 @@ const Index = ({
     const searchStatusComponents = () => (
         <>
             {searchStatus === searchPageStatus.SHOW_RESULTS && (
-                <SearchResults availableSearchOptions={availableSearchOptions} fallbackTriggered={fallbackTriggered} />
+                <SearchResults
+                    availableSearchOptions={availableSearchOption}
```

Comment:
Hey,
Does it make sense to rename `availableSearchOptions` to `availableSearchOption` in order to maintain the consistency of the variable name across different components? 
Thanks!